hack/build-and-push.sh

```diff
@@ -77,10 +77,16 @@ docker build --build-arg COMMIT=${BINARY_VERSION} --build-arg TAG=${VERSION_TAG}
 docker push ${REGISTRY}/admission-server:${GIT_TAG}
 
 # Then, we add extra tags if required.
+# If the version tag and the git tag aren't the same, we're on a release branch, so
+# we need to push the release tag.
 if [[ $VERSION_TAG != $GIT_TAG ]]
 then
     docker tag ${REGISTRY}/admission-server:${GIT_TAG} ${REGISTRY}/admission-server:${VERSION_TAG}
     docker push ${REGISTRY}/admission-server:${VERSION_TAG}
+else
+# Otherwise, we're on master and we should update the master image here too.
+    docker tag ${REGISTRY}/admission-server:${GIT_TAG} ${REGISTRY}/admission-server:master
```

Comment:
Since we're planning to move to "main", should we get ahead of it here? Alternatively should we just use "latest"?

Reply:
Yeah, making this PR is what reminded me about moving to `main`. You're right that `latest` is probably better used for a nightly-style thing anyway. I'll update to that effect.